app/controller.py

```diff
@@ -188,3 +195,51 @@ def troll_katie():
         turn_light_off()
         sleep(0.13)
     return redirect(('/' + get_current_day_name()).lower())
+
+"""
+Made initially for testing sound.
+Use it now and then to check if everything working properly.
+"""
+@app.route('/sound')
+def yolo_def():
```

Comment:
I think the function name should describe what the function does; how about `play_sound` instead?